src/app/page.tsx

```diff
@@ -3,26 +3,43 @@
 import { LiveImageShapeUtil } from "@/components/live-image";
 import * as fal from "@fal-ai/serverless-client";
 import { Editor, Tldraw } from "@tldraw/tldraw";
+import { useCallback } from "react";
 
 fal.config({
   requestMiddleware: fal.withProxy({
     targetUrl: "/api/fal/proxy",
   }),
 });
 
+const shapeUtils = [LiveImageShapeUtil];
```

Comment:
There's a re-render error that sometimes happens if you don't pull out the `shapeUtils` prop like this. We fixed it a while back, but looks like the issue is happening again. Even after doing this, I'm still getting some re-render errors though 🤔

Reply:
Ah, interesting. I was puzzled by that myself, this explains things. Thanks :)